tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -1486,6 +1486,22 @@ public void MockDirectory_EnumerateDirectories_ShouldThrowWhenPathIsNotMocked()
             // Assert
             Assert.Throws<DirectoryNotFoundException>(action);
         }
+        
+        
+        [TestCase("Folder", "Folder/SubFolder")]
+        [TestCase("/Folder", "/Folder/SubFolder")]
+        [TestCase("Folder/", "Folder/SubFolder")]
```

Comment:
In the real file system this returns an empty collection and not one that contains "Folder/SubFolder".

Reply:
Sorry, but I cannot reproduce your finding. I've checked both on GNU/Linux and Windows: Neither OS cares about the trailing path separator.

Reply:
I created a repro-branch here: https://github.com/Testably/Testably.Abstractions/pull/415/files

Reply:
Sorry, that was my error. Looks good :-)